src/CodeExamples.sln

```diff
@@ -23,6 +23,40 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMap", "HashMap\HashMap.
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "TestHashMap", "TestHashMap\TestHashMap.csproj", "{EBDC072A-4291-4482-9BAE-2490C190B1B6}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMapNamespace.CalculateUintCapacity", "HashMap.Test\HashMapNamespace.CalculateUintCapacity\HashMapNamespace.CalculateUintCapacity.csproj", "{4DC0AECB-322A-427A-B49B-FA54AF5A8D16}"
```

Comment:
### 2/4 Tests Pass
```
vscode ➜ .../CodeExamples/src/HashMap.Test/HashMapNamespace.CalculateUintCapacity (roost-1709036480) $ dotnet test
  Determining projects to restore...
  Restored /workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.CalculateUintCapacity/HashMapNamespace.CalculateUintCapacity.csproj (in 476 ms).
  1 of 2 projects are up-to-date for restore.
  HashMap -> /workspaces/CodeExamples/src/HashMap/bin/Debug/net6.0/HashMap.dll
/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.CalculateUintCapacity/CalculateUintCapacity.cs(96,13): warning NUnit2005: Consider using the constraint model, Assert.That(actual, Is.EqualTo(expected)), instead of the classic model, Assert.AreEqual(expected, actual) (https://github.com/nunit/nunit.analyzers/tree/master/documentation/NUnit2005.md) [/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.CalculateUintCapacity/HashMapNamespace.CalculateUintCapacity.csproj]
/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.CalculateUintCapacity/CalculateUintCapacity.cs(111,13): warning NUnit2005: Consider using the constraint model, Assert.That(actual, Is.EqualTo(expected)), instead of the classic model, Assert.AreEqual(expected, actual) (https://github.com/nunit/nunit.analyzers/tree/master/documentation/NUnit2005.md) [/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.CalculateUintCapacity/HashMapNamespace.CalculateUintCapacity.csproj]
/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.CalculateUintCapacity/CalculateUintCapacity.cs(125,13): warning NUnit2005: Consider using the constraint model, Assert.That(actual, Is.EqualTo(expected)), instead of the classic model, Assert.AreEqual(expected, actual) (https://github.com/nunit/nunit.analyzers/tree/master/documentation/NUnit2005.md) [/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.CalculateUintCapacity/HashMapNamespace.CalculateUintCapacity.csproj]
  HashMapNamespace.CalculateUintCapacity -> /workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.CalculateUintCapacity/bin/Debug/net7.0/HashMapNamespace.CalculateUintCapacity.dll
Test run for /workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.CalculateUintCapacity/bin/Debug/net7.0/HashMapNamespace.CalculateUintCapacity.dll (.NETCoreApp,Version=v7.0)
Microsoft (R) Test Execution Command Line Tool Version 17.7.2 (x64)
Copyright (c) Microsoft Corporation.  All rights reserved.

Starting test execution, please wait...
A total of 1 test files matched the specified pattern.
  Failed TestCalculateUintCapacity_HigherInitCapacity [65 ms]
  Error Message:
     Expected: 267
  But was:  266

  Stack Trace:
     at HashMapNamespace.Test.CalculateUintCapacityTests.TestCalculateUintCapacity_HigherInitCapacity() in /workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.CalculateUintCapacity/CalculateUintCapacity.cs:line 111

  Failed TestCalculateUintCapacity_LowerInitCapacity [3 ms]
  Error Message:
     Expected: 3
  But was:  2

  Stack Trace:
     at HashMapNamespace.Test.CalculateUintCapacityTests.TestCalculateUintCapacity_LowerInitCapacity() in /workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.CalculateUintCapacity/CalculateUintCapacity.cs:line 96


Failed!  - Failed:     2, Passed:     2, Skipped:     0, Total:     4, Duration: 78 ms - HashMapNamespace.CalculateUintCapacity.dll (net7.0)
vscode ➜ .../CodeExamples/src/HashMap.Test/HashMapNamespace.CalculateUintCapacity (roost-1709036480) $
```

Reply:
These tests fail in expectation due to the behavior of SMath.Round()
It is a minor gap.